### DISCUSSION TITLE
Would it be possible to port openrc-settingsd to run on Void/runit?

### DISCUSSION THREAD

**Discussion by mayhair** [void-linux/void-packages#48923] [Q&A]
GNOME and other desktop environments have graphical options to change the time zone, hostname, etc. They use dbus interfaces which are packaged with systemd (for example, changing the time zone relies on [`org.freedesktop.timedate1`](https://www.freedesktop.org/software/systemd/man/latest/org.freedesktop.timedate1.html)). This means that by default, these options do not work on distros that do not use systemd, including Void.

Gentoo and Alpine, which use OpenRC, are already using [`openrc-settingsd`](https://gitlab.com/postmarketOS/openrc-settingsd) to make these options work. Void uses runit, so it cannot use this service as-is, but I still wonder if it would be easy to port it.

I know that these options can already be changed via the command-line, but it would be nice to implement this for users who prefer to change settings graphically.

**Comment by classabbyamp** [comment]
> hostnamed

void generally doesn't use `/etc/machine-info`, but I guess it would work as-is.

> localed

void doesn't use `/etc/vconsole.conf` so this would not work as-is.

> timedated

I've been [working on something for this](https://github.com/classabbyamp/sundial), but it's not done.

**Reply by pondichys** [reply]
For the **localed** functionalities, I created a PR for `blocaled` that is used in Artix linux.
See #51957
